xchange-core/src/test/java/org/knowm/xchange/dto/account/BalanceTest.java

```diff
@@ -0,0 +1,28 @@
+package org.knowm.xchange.dto.account;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import org.junit.Test;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.utils.ObjectMapperHelper;
+
+public class BalanceTest {
+
```

Comment:
please add one test that will allow us to check if Currency is serialized as a String:

```

  @Test
  public void testSerialize() throws IOException {
    Balance balanse = ...;
    String serialized = ObjectMapperHelper.toCompactJSON(balanse);
    assertThat(balanse).contains("\"currency\":\"ADA\"");
  }
```